manifests/base/cuda-rstudio-buildconfig.yaml

```diff
@@ -120,10 +120,10 @@ spec:
   resources:
     limits:
       cpu: "1500m"
-      memory: 4Gi
+      memory: 8Gi
     requests:
       cpu: "1500m"
-      memory: 4Gi
+      memory: 8Gi
```

Comment:
technically, it can't really guarantee ephemeral-storage
however, norms are that a node having more memory would have the ephemeral-storage too
and as the build wouldn't use the entire space, that would also be used by the file system.
this buffer should help out.

Reply:
Probably won't help me, I have Single Node Openshift cluster. But I'll try today

Reply:
Iiuc the error message, my node has 20 gb ephemeral storage and it kills the build after 6gb of storage is used. Why is it so aggressive?

Reply:
> technically, it can't really guarantee ephemeral-storage

Do you have more details on that? https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage does not set it?

> Iiuc the error message, my node has 20 gb ephemeral storage and it kills the build after 6gb of storage is used. Why is it so aggressive?

That's what the error message seems to say, but looking at disk space, following https://docs.openshift.com/container-platform/4.14/storage/understanding-ephemeral-storage.html I took some "real" disk usage measurements, that I'm noting down at

* https://github.com/red-hat-data-services/notebooks/pull/157